roboflow/core/version.py

```diff
@@ -300,24 +300,32 @@ def deploy(self, model_type: str, model_path: str) -> None:
                 )
             )
 
-        try:
-            import torch
-            import ultralytics
-        except ImportError as e:
-            raise (
-                "The ultralytics python package is required to deploy yolov8 models. Please install it with `pip install ultralytics`"
-            )
-
-        # add logic to save torch state dict safely
         if model_type == "yolov8":
-            model = torch.load(os.path.join(model_path + "weights/best.pt"))
+            try:
+                import torch
+                import ultralytics
+            except ImportError as e:
+                raise (
+                    "The ultralytics python package is required to deploy yolov8 models. Please install it with `pip install ultralytics`"
+                )
+        elif model_type == "yolov5":
+            try:
+                import torch
+            except ImportError as e:
+                raise (
+                    "The torch python package is required to deploy yolov5 models. Please install it with `pip install torch`"
+                )
+
+        model = torch.load(os.path.join(model_path, "weights/best.pt"))
 
-            class_names = []
-            for i, val in enumerate(model["model"].names):
-                class_names.append((val, model["model"].names[val]))
-            class_names.sort(key=lambda x: x[0])
-            class_names = [x[1] for x in class_names]
+        class_names = []
+        for i, val in enumerate(model["model"].names):
+            class_names.append((val, model["model"].names[val]))
+        class_names.sort(key=lambda x: x[0])
+        class_names = [x[1] for x in class_names]
 
```

Comment:
I know this is not for this PR, but could we add comments about what is going on for yolov8?